src/app/codegen-data-model/tests/TestCodegenModelViaMocks.cpp

```diff
@@ -2430,6 +2435,41 @@ TEST(TestCodegenModelViaMocks, EmberWriteAttributeAccessInterfaceTest)
     TestEmberScalarNullWrite<int64_t, ZCL_INT64S_ATTRIBUTE_TYPE>();
 }
 
+TEST(TestCodegenModelViaMocks, EmberInvokeTest)
+{
+    // Ember invoke is extrealy hard-coded, so generally we just need to validate that
+    // correct paths are invoked.
+
+    UseMockNodeConfig config(gTestNodeConfig);
+    chip::app::CodegenDataModel model;
+
+    {
+        const ConcreteCommandPath kCommandPath(kMockEndpoint1, MockClusterId(1), kMockCommandId1);
+        const InvokeRequest kInvokeRequest{ .path = kCommandPath };
+        chip::TLV::TLVReader tlvReader;
+
+        const uint32_t kDispatchCountPre = chip::Test::DispatchCount();
+
+        ASSERT_EQ(model.Invoke(kInvokeRequest, tlvReader, nullptr /* handler, NOT used by impl*/), CHIP_NO_ERROR);
```

Comment:
nit, Is this coding style allowed? I think I have seem something like this more often
```suggestion
        // Using handler set to nullptr as it is not used by impl
        ASSERT_EQ(model.Invoke(kInvokeRequest, tlvReader, /*handler=*/ nullptr), CHIP_NO_ERROR);
```